.dotnet/src/Custom/FineTuning/FineTuningClient.cs

```diff
@@ -62,4 +64,50 @@ protected internal FineTuningClient(ClientPipeline pipeline, Uri endpoint, OpenA
         _pipeline = pipeline;
         _endpoint = endpoint;
     }
+
+    /// <summary> Creates a job with a training file and model. </summary>
+    /// <param name="training_file"> The training file name that is already uploaded. String should match pattern '^file-[a-zA-Z0-9]{24}$'  </param>
+    /// <param name="model"> The model name to fine-tune. String such as "gpt-3.5-turbo" </param>
+    /// <param name="options"> Additional options (<see cref="RequestOptions"/>) to customize the request. </param>
+    public ClientResult<FineTuningJob> CreateJob(string training_file, string model, RequestOptions options = default)
+    {
+        var request = new InternalCreateFineTuningJobRequest(model, training_file);
+        var content = request.ToBinaryContent();
+        ClientResult result = CreateJob(content, options);
+        return ClientResult.FromValue(FineTuningJob.FromResponse(result.GetRawResponse()), result.GetRawResponse());
+    }
+
+    /// <summary> Async version of create job</summary>
+    /// <param name="training_file"> The training file name that is already uploaded. String should match pattern '^file-[a-zA-Z0-9]{24}$'  </param>
+    /// <param name="model"> The model name to fine-tune. String such as "gpt-3.5-turbo" </param>
+    /// <param name="options"> Additional options (<see cref="RequestOptions"/>) to customize the request. </param>
+    public async Task<ClientResult<FineTuningJob>> CreateJobAsync(string training_file, string model, RequestOptions options = default)

```

Comment:
hinted at above, but all of these public client methods should include the "virtual" keyword to allow derived overrides